impl/src/main/java/org/ehcache/impl/internal/store/offheap/AbstractOffHeapStore.java

```diff
@@ -1117,6 +1058,29 @@ private void notifyInvalidation(final K key, final ValueHolder<V> p) {
     }
   }
 
+  private OffHeapValueHolder<V> computeWithRetry(K key, BiFunction<K, OffHeapValueHolder<V>, OffHeapValueHolder<V>> computeFunction, boolean fault) throws StoreAccessException {
```

Comment:
I wonder why `computeWithRetry` only tries to retry once on `OversizeMappingException` while the old code used to rety in a `while (true)` loop. Can you elaborate?